crates/interpreter/src/instructions/bitwise.rs

```diff
@@ -103,19 +104,560 @@ pub fn sar<H: Host, SPEC: Spec>(interpreter: &mut Interpreter, _host: &mut H) {
+            SHL
+            ---
+            0x8000000000000000000000000000000000000000000000000000000000000000
+            */
+            TestCase {
+                value: "0xffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffff",
+                shift: "0xff",
+                expected: "0x8000000000000000000000000000000000000000000000000000000000000000",
+            },
+            /*
+            PUSH 0xffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffff
+            PUSH 0x0100
+            SHL
+            ---
+            0x0000000000000000000000000000000000000000000000000000000000000000
+            */
+            TestCase {
+                value: "0xffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffff",
+                shift: "0x0100",
+                expected: "0x0000000000000000000000000000000000000000000000000000000000000000",
+            },
+            /*
+            PUSH 0x0000000000000000000000000000000000000000000000000000000000000000
+            PUSH 0x01
+            SHL
+            ---
+            0x0000000000000000000000000000000000000000000000000000000000000000
+            */
+            TestCase {
+                value: "0x0000000000000000000000000000000000000000000000000000000000000000",
+                shift: "0x01",
+                expected: "0x0000000000000000000000000000000000000000000000000000000000000000",
+            },
+            /*
+            PUSH 0x7fffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffff
+            PUSH 0x01
+            SHL
+            ---
+            0xfffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffe
+            */
+            TestCase {
+                value: "0x7fffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffff",
+                shift: "0x01",
+                expected: "0xfffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffe",
+            },
+        ];
+
+        for test in test_cases {
+            host.clear();
+            push!(interpreter, U256::from_str(test.value).unwrap());
+            push!(interpreter, U256::from_str(test.shift).unwrap());
+            shl::<DummyHost, LatestSpec>(&mut interpreter, &mut host);
+            pop!(interpreter, res);
+            assert_eq!(res, U256::from_str(test.expected).unwrap());
+        }
+    }
+
+    #[test]
+    fn test_logical_shift_right() {
+        let mut host = DummyHost::new(Env::default());
+        let mut interpreter = Interpreter::new(Contract::default(), u64::MAX, false);
+
+        struct TestCase {
+            value: &'static str,
+            shift: &'static str,
+            expected: &'static str,
+        }
+
+        let test_cases = [
+            /*
+            PUSH 0x0000000000000000000000000000000000000000000000000000000000000001
+            PUSH 0x00
+            SHR
+            ---
+            0x0000000000000000000000000000000000000000000000000000000000000001
+            */
```

Comment:
Comment with value looks redundant

Reply:
Yes thanks, it's much cleaner without these.